conf/defaults.ini

```diff
@@ -874,6 +874,10 @@ list_metrics_page_limit = 500
 # Experimental, for use in Grafana Cloud only. Please do not set.
 external_id =
 
+# Set the plugins that will receive AWS settings for each request (via plugin context)
+# By default this will include all Grafana Labs owned AWS plugins, or those that make use of AWS settings (ElasticSearch, Prometheus).
+forward_settings_to_plugins = cloudwatch, grafana-athena-datasource, grafana-redshift-datasource, grafana-x-ray-datasource, grafana-timestream-datasource, grafana-iot-sitewise-datasource, grafana-iot-twinmaker-app, grafana-opensearch-datasource, aws-datasource-provisioner, elasticsearch, prometheus
```

Comment:
I added all the external aws plugins (even though they're not using the context yet) and ElasticSearch and Prometheus (which use sigv4) to the list along with CloudWatch